core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1037,6 +1071,32 @@ private void sendNotificationBroadcast(int type, int code) {
         sendBroadcast(intent);
     }
 
+    private void skipIfMetEnding() {
+        Playable playable = mediaPlayer.getPlayable();
+        if (! (playable instanceof FeedMedia)) { return; }
+
+        int duration = getDuration();
+        int remainingTime = duration - getCurrentPosition();
+
+        FeedMedia feedMedia = (FeedMedia) playable;
+        FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
+        int skipEnd = preferences.getFeedSkipEnding();
+        if (skipEnd > 0 && remainingTime < skipEnd * 1000) {
+            Log.d(TAG, "skipIfMetEnding: Skipping the remaining " + remainingTime / 1000 + " " + skipEnd );
+            Context context = getApplicationContext();
+            String skipMesg = String.format(context.getString(R.string.pref_feed_skip_ending_toast),
```

Comment:
I think you can directly add the format parameters to getString. Then Android Studio can do better analysis.

Reply:
fixed